input/profiles/StructureDefinition-cqm-testcase.json

```diff
@@ -222,6 +245,29 @@
       {
         "id": "MeasureReport.group.stratifier.stratum.population",
         "path": "MeasureReport.group.stratifier.stratum.population",
+        "constraint": [
+          {
+            "key": "cntq-1",
+            "severity": "error",
+            "human": "Either count or countQuantity must be provided, but not both.",
+            "expression": "(count.exists() xor extension('http://hl7.org/fhir/StructureDefinition/measurereport-countQuantity').exists())"
+          }
+        ],
+        "mustSupport": true
+      },
+      {
+        "id": "MeasureReport.group.stratifier.stratum.population.extension:countQuantity",
+        "path": "MeasureReport.group.stratifier.stratum.population.extension",
+        "min": 0,
```

Comment:
Please add a slice name such as "countQuantity". Right now, the element only displayed as "extension".